Runtime/MeshOperations/AppendElements.cs

```diff
@@ -250,9 +260,18 @@ public static ActionResult CreateShapeFromPolygon(this ProBuilderMesh mesh, ILis
             {
                 int[] indexes = triangles.ToArray();
 
+                // check that all points are represented in the triangulation
+                HashSet<int> triangleSet = new HashSet<int>(triangles);
```

Comment:
Would it make sense to have a unit test to check this condition?

Reply:
It would be nice if we could pass some parameter to the triangulation function instructing it to fail on the omission of a control point, but I don't see a quick way to make that work. Is there another way we can validate that the number of points triangulated matches the input without initializating a HashSet every time? (If not that's okay, I'm just asking)

Reply:
@karl- I updated it to check the distinctIndices instead of creating a hashset.